packages/amplify_api/example/integration_test/graphql_tests.dart

```diff
@@ -112,10 +115,11 @@ void main() {
       await deleteAllBlogsAndPosts();
     });
 
-    testWidgets('should fetch', (WidgetTester tester) async {
-      const listBlogs = 'listBlogs';
-      const items = 'items';
-      String graphQLDocument = '''query MyQuery {
+    group('queries', () {
```

Comment:
adding these groups (and indentation) caused the diff to explode. Sorry about that. Only subscriptions really changed here (and added a field to create helper selection set).